README/CREDITS

```diff
@@ -928,6 +928,13 @@ N: Tomasz Sosnicki
 E: tom.sosnicki@gmail.com
 D: contribution to gviz3d/TStructViewer
 
+N: Thibault Souquet
+E: contact@thibaultsouquet.fr
+D: Image viewer for the ROOT Browser (v7)
+D: Breadcrumbs display for the ROOT Browser (v7)
+D: ROOT Browser (v7) drawing options
+D: Improvements in JavaScript ROOT GUI
+
```

Comment:
You may also add:
D: Improvements in the ROOT website Jekyll theme.

Reply:
Bien vu, merci Olivier!